app/src/main/java/org/jellyfin/androidtv/querying/StdItemQuery.java

```diff
@@ -5,13 +5,15 @@
 import org.jellyfin.apiclient.model.querying.ItemFields;
 import org.jellyfin.apiclient.model.querying.ItemQuery;
 
-/**
- * Created by Eric on 12/5/2014.
- */
 public class StdItemQuery extends ItemQuery {
-
     public StdItemQuery(ItemFields[] fields) {
-        if (fields == null) fields = new ItemFields[] {ItemFields.PrimaryImageAspectRatio, ItemFields.Overview, ItemFields.ItemCounts, ItemFields.DisplayPreferencesId};
+        if (fields == null) fields = new ItemFields[] {
```

Comment:
I think it would look better to use curly brackets because the assignment is multi-line.

Reply:
Good catch. It's fixed now. (I actually prefer always using curly brackets.)